vars/protecodeExecuteScan.groovy

```diff
@@ -0,0 +1,135 @@
+import com.sap.piper.GenerateDocumentation
+import com.sap.piper.JenkinsUtils
+import com.sap.piper.PiperGoUtils
+import com.sap.piper.Utils
+
+import groovy.transform.Field
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field String STEP_NAME = getClass().getName()
+@Field String METADATA_FILE = 'metadata/protecode.yaml'
+@Field Set GENERAL_CONFIG_KEYS = [
```

Comment:
`GENERAL_CONFIG_KEYS` can be removed completely.
Information is retrieved from `metadata/protecode.yaml`

Reply:
You will have to edit CommonStepTest and add your step to the excludes, otherwise you get an issue upon execution.